test/e2e/kubevirt.go

```diff
@@ -1267,8 +1278,11 @@ passwd:
 
 			if td.topology == "localnet" {
 				By("setting up the localnet underlay")
-				nodes := ovsPods(clientSet)
-				Expect(nodes).NotTo(BeEmpty())
+				err, nodes := ovsPods(clientSet)
+				Expect(err).To(Succeed())
+				if len(nodes) == 0 {
+					Skip("No ovs pods: topology localnet test skipped")
+				}
```

Comment:
I actually do wonder if we want to skip these tests ... 

Are you doing this so you can e2e test against deployments running ovs in the nodes ? 
If so, I'd say the long term solution would be to tap into the node based OVS instead ...

Can be done in a follow up, of course.

Reply:
yeah, that is it. I would like to address that as a follow up pr. but the long term fix is really to modify the functions to make the ovs changes in the node.